helmchart/sdp/values.yaml

```diff
@@ -297,44 +418,72 @@ tss:
+  ## @param tss.configMap.data.MAX_BASE_FEE Specifies the maximum base fee (in stroops) the TSS is willing to pay per transaction. This helps to control costs and ensures transactions are economically feasible.
+  ## @param tss.configMap.data.TSS_METRICS_TYPE Defines the type of metrics system that the TSS should use. Options: "TSS_PROMETHEUS".
   configMap:
-    # Annotations to add to the ConfigMap
     annotations:
-
     data:
       NUM_CHANNEL_ACCOUNTS: "1"
       MAX_BASE_FEE: "100000"
-      MOCK: "false"
       TSS_METRICS_TYPE: "TSS_PROMETHEUS"
 
-  # =========================== START tss.kubeSecrets ===========================
+  ## @extra tss.kubeSecrets Kubernetes secrets are used to manage sensitive information, such as API keys and private keys. It's crucial that these details are kept private.
+  ## @param tss.kubeSecrets.secretName The name of the Kubernetes secret object. Only use this if create is false.
+  ## @param tss.kubeSecrets.create If true, the secret will be created. If false, it is assumed the secret already exists.
+  ## @param tss.kubeSecrets.annotations Annotations to be added to the secret.
+  ## @param tss.kubeSecrets.data.DATABASE_URL URL of the database used by the TSS.
+  ## @param tss.kubeSecrets.data.DISTRIBUTION_SEED The private key of the Stellar account used to disburse funds.
   kubeSecrets:
     secretName: "tss-secret-name"
-    # Create the secret for tss
     create: false
-
-    # Annotations to add to the secret
     annotations:
-
-    # The data to be stored in the secret
     data:
+      DATABASE_URL: #required
       DISTRIBUTION_SEED: #required
 
-# =========================== FRONTEND DASHBOARD ===========================
+
+# =========================== START dashboard ===========================
+## @section Dashboard
+## @descriptionStart
+## Configuration parameters for the Dashboard. This is the user interface administrators use to initiate and track the progress of disbursements.
+## @descriptionEnd
 dashboard:
+
+  ## @param dashboard.enabled If true, the dashboard will be deployed.
   enabled: false
+
+  ## @extra dashboard.route Configuration related to the routing of the Dashboard.
+  ## @param dashboard.route.schema Protocol scheme used for the service. Can be "http" or "https".
+  ## @param dashboard.route.domain Public domain/address of the Dashboard.
+  ## @param dashboard.route.port Primary port on which the Dashboard listens.
   route:
     schema: "https"
     domain: "sdp-dashboard.localhost.com"
     port: "80"
+
+  ## @extra dashboard.image Configuration related to the Docker image used by the Dashboard.
+  ## @param dashboard.image.fullName Full name of the Docker image.
+  ## @param dashboard.image.pullPolicy Image pull policy for the dashboard. For locally built images, consider using "Never" or "IfNotPresent".
   image:
     fullName: stellar/stellar-disbursement-platform-frontend:latest
     pullPolicy: IfNotPresent
```

Comment:
Same here: when using `latest`, we should default to `pullPolicy: Always` to ensure the latest image will be pulled.